chord_metadata_service/mohpackets/api_discovery.py

```diff
@@ -310,14 +321,14 @@ def diagnosis_age_count(_request):
         donor = date["submitter_donor_id"]
         cur_date = (
             dt.strptime(date["date_of_diagnosis"], "%Y-%m").date()
-            if date["date_of_diagnosis"] != ""
-            else ""
+            if date["date_of_diagnosis"] is not None
+            else None
         )
         if donor not in min_dates.keys():
             min_dates[donor] = cur_date
         else:
-            if (min_dates[donor] != "" and cur_date < min_dates[donor]) or (
-                min_dates[donor] == "" and cur_date != ""
+            if (min_dates[donor] is not None and cur_date < min_dates[donor]) or (
```

Comment:
I got this error when running `diagnosis_age_count` : 
```
TypeError at /v2/discovery/overview/diagnosis_age_count
'<' not supported between instances of 'NoneType' and 'datetime.date'
```
I guess that when `cur_date` is None and `min_dates[donor]` has a value, it can't do the comparision

Reply:
Good catch, should be fixed now